src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DefaultForegroundDispatcher.cs

```diff
@@ -5,24 +5,25 @@
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Razor;
 
-namespace Microsoft.AspNetCore.Razor.LanguageServer.Common
+namespace Microsoft.CodeAnalysis.Razor.Workspaces
 {
     internal class DefaultForegroundDispatcher : ForegroundDispatcher
```

Comment:
Yeah I agree that `Foreground` doesn't seem to capture the intent here anymore. Maybe something like `SerialDispatcher` or `SingleThreadedDispatcher`? `Serial` might be a no-go because it's a bit of an overload with the O# Serial vs Parallel concepts.